test/extensions/transport_sockets/tls/integration/BUILD

```diff
@@ -41,3 +39,19 @@ envoy_cc_test(
         "@envoy_api//envoy/extensions/transport_sockets/tls/v3:pkg_cc_proto",
     ],
 )
+
+envoy_cc_test(
```

Comment:
shall we move this to envoy_extension_cc_test in corresponding cert validator extension directory?

Reply:
Thanks!